x-pack/plugin/otel-data/README.md

```diff
@@ -0,0 +1,44 @@
+NOTE: this plugin is not related to APM Metrics used in ES codebase. The APM Metrics are in :modules:apm
+
+## OpenTelemetry Ingest plugin
+
+The OpenTelemetry Ingest plugin installs index templates and component templates for Elastic OpenTelemetry data.
+
+All resources are defined as YAML under [src/main/resources](src/main/resources).
+
+The OpenTelemetry index templates rely on mappings from `x-pack-core`.
+See [x-pack/plugin/core/src/main/resources](../core/src/main/resources).
+
+## Adding/Removing/Updating a resource
+
+All resources are defined as YAML under [src/main/resources](src/main/resources).
+
+For a resource to be known to the plugin it must be added to
+[src/main/resources/resources.yaml](src/main/resources/resources.yaml) in the
+appropriate section.
+
+Any update to resources included by this package also requires a bump to the
+`version` property included in the resources file.
+
+## Testing
+
+## Unit testing
+
+Java unit tests cover basic, low-level details of the plugin, such as the parsing and loading of resources.
+These can be run with:
+
+```
+./gradlew x-pack:plugin:apm-data:test
```

Comment:
why are we referencing unit tests in this plugins readme but tests live in apm-data?

Reply:
We don't. Totally my fault, this was some leftover. 

I noticed just right after I flipped this PR to ready-to-review, then I pushed https://github.com/elastic/elasticsearch/pull/111091/commits/1c6192990883db59473710541e08489c679f33b2 - probably the same time as you looked at it. Sorry about this.